libr/parse/parse.c

```diff
@@ -262,17 +262,13 @@ static int filter(RParse *p, ut64 addr, RFlag *f, char *data, char *str, int len
 			}
 			if (f) {
 				RFlagItem *flag2;
-				flag = r_flag_get_i2 (f, off);
+				flag = r_flag_get_by_spaces (f, off, "symbols", "imports",
```

Comment:
i think you forgot `"functions"` before everything else.

Reply:
yep. can you add this?  i'll merge after that